packages/api/app/Controllers/Http/TechnologyController.js

```diff
@@ -55,8 +55,11 @@ class TechnologyController {
 	 * GET technologies/:id
 	 */
 	async show({ request }) {
+		const filters = request.all();
+
 		return Technology.query()
 			.withParams(request.params)
+			.withFilters(filters, request.params)
```

Comment:
Não é uma alteração redundante? O `request.all()` faz o merge de query e body params.

```suggestion
			.withFilters(filters)
```